### ISSUE TITLE
Better CLI commands

### ISSUE THREAD

**Issue by dakom** [Lay3rLabs/WAVS#195]
- DeployCore
  - just deploys the core contracts
  - prints env vars when done

- DeployService
  - takes core contracts (set in cli args or env vars)
  - deploys service contracts
  - prints env vars when done

- DeployAll
  - basically calls DeployCore and then DeployService
 
- AddTask (new command)
  - Hardcode HelloWorld for now
  - "wavs" boolean
    - if false, just submits result right to chain (current functionality)
    - if true, does full e2e test
 
- KitchenSink
  - basically a combo of DeployAll and AddTask
